it/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,51 +33,51 @@
-WalletNotification = Potete salvare i dati inseriti nelle form in modo che vengano inseriti automaticamente in futuro. Per salvare queste informazioni selezionate Modifica/Registra Dati Form dal menu mentre visualizzate la form.
-noPrefills = Non ci sono campi da riempire automaticamente.
+newPassword = Seleziona una nuova Password principale.#(Se non desideri utilizzare una Password principale lasciare i campi in bianco).#
+enterPassword = Inserisci una Password principale
+confirmPassword = Digita di nuovo la Password principale
+confirmFailed_TryAgain? = Le due Password non corrispondono.  Si prega di inserire la stessa Password principale in entrambi i campi.
+SelectUser = Seleziona un nome utente da inserire in questo modulo.
+SelectUserWhosePasswordIsBeingChanged = Seleziona l'utente al quale modificare la password.
+PasswordNotification = Per comodità, il browser può ricordare il tuo nome utente e la tua password, così non dovrai inserirli di nuovo in futuro. La tua password sarà oscurata prima di essere salvata sul tuo disco fisso. Vuoi abilitare questa funzione?
+WalletNotification = Puoi salvare i dati inseriti nei moduli in modo che vengano compilati automaticamente in altri moduli in futuro. Per salvare questi dati seleziona Modifica/Salva dati modulo dal menu durante la visualizzazione del modulo.
+noPrefills = Non sono presenti campi che possono essere compilati automaticamente.
 Never = Mai per questo sito
-Confirm = Confirm
+Confirm = Conferma
 PromptForPassword = Password
 PromptForData = Prompt
-SaveTheseValuesObscured = Usate il Gestore Password per ricordare questi dati.
-SaveTheseValuesEncrypted = Usate il Gestore Password per ricordare questi dati.
-SaveThisValueObscured = Utilizzare Gestione password per ricordare questo valore.
-SaveThisValueEncrypted = Utilizzare Gestione password per ricordare questo valore.
-SaveThisPasswordObscured= Utilizzare Gestione password per memorizzare questa password.
-SaveThisPasswordEncrypted = Utilizzare Gestione password per memorizzare questa password.
-xSaveTheseValuesObscured = Ricorda questi dati. I dati registrati non sono protetti da password.
-xSaveTheseValuesEncrypted = Ricorda questi dati. I dati registrati non sono protetti da password.
-xSaveThisValueObscured = Remember this value.  Stored values are not password protected.
-xSaveThisValueEncrypted = Remember this value.  Stored values are password protected.
+SaveTheseValuesObscured = Utilizza il Gestore password per ricordare questi dati.
+SaveTheseValuesEncrypted = Utilizza il Gestore password per ricordare questi dati.
+SaveThisValueObscured = Utilizza il Gestore password per ricordare questo valore.
+SaveThisValueEncrypted = Utilizza il Gestore password per ricordare questo valore.
+SaveThisPasswordObscured= Utilizza il Gestore password per ricordare questa password.
+SaveThisPasswordEncrypted = Utilizzare Gestione password per ricordare questa password.
```

Comment:
CORRECTIONS NEEDED: In order to keep the text consistent, I would use "Utilizza" instead of "Utilizzare" in this sentence, as follows: 

Utilizza Gestione password per ricordare questa password.